letsencrypt/tests/cli_test.py

```diff
@@ -30,19 +32,19 @@ def setUp(self):
         self.config_dir = os.path.join(self.tmp_dir, 'config')
         self.work_dir = os.path.join(self.tmp_dir, 'work')
         self.logs_dir = os.path.join(self.tmp_dir, 'logs')
+        self.standard_args = ['--text', '--config-dir', self.config_dir,
+            '--work-dir', self.work_dir, '--logs-dir', self.logs_dir,
+            '--agree-dev-preview']
 
     def tearDown(self):
         shutil.rmtree(self.tmp_dir)
 
     def _call(self, args):
-        from letsencrypt import cli
-        args = ['--text', '--config-dir', self.config_dir,
-                '--work-dir', self.work_dir, '--logs-dir', self.logs_dir,
-                '--agree-dev-preview'] + args
+        args = self.standard_args + args
         with mock.patch('letsencrypt.cli.sys.stdout') as stdout:
             with mock.patch('letsencrypt.cli.sys.stderr') as stderr:
                 with mock.patch('letsencrypt.cli.client') as client:
-                    ret = cli.main(args)
+                    ret = cli.main(args[:]) # NOTE: parser can alter its args!
```

Comment:
I guess it is unnecessary because [this line](https://github.com/letsencrypt/letsencrypt/pull/1455/files#diff-1333bfbf9a96ab6aa2587c1ca0cbd2b3R43) protects the caller's copy of the list from being modified in place.  I still find that a terrifying side effect...